Library/Formula/hue.rb

```diff
@@ -0,0 +1,99 @@
+class Hue < Formula
+  desc "Hue is a Web interface for analyzing data with Apache Hadoop."
+  homepage "http://gethue.com/"
+  url "https://github.com/cloudera/hue/archive/release-3.8.1.tar.gz"
+  sha256 "582777f567b9f4a34e4ce58cfd5ed24aff15f3e02f193e7990329d8cb0161a8a"
+
+  depends_on :java
+  depends_on :mysql
+
+  depends_on "gmp"
+  depends_on "maven" => :build
+  depends_on "openssl"
+
+  patch :DATA
+
+  def install
+    ENV.append_to_cflags "-I" + `xcrun --show-sdk-path`.strip + "/usr/include/sasl" if MacOS.version >= :mavericks
+    ENV.deparallelize
```

Comment:
@mikemcquaid Just to make sure I understood which line you are referring to - are you asking if I can file an upstream issue on the build not working in parallel?